packages/backend/server/src/core/quota/schema.ts

```diff
@@ -174,6 +174,27 @@ export const Quotas: Quota[] = [
       copilotActionLimit: 10,
     },
   },
+  {
+    feature: QuotaType.TeamPlanV1,
+    type: FeatureKind.Quota,
+    version: 1,
+    configs: {
+      // quota name
+      name: 'Team Workspace',
+      // single blob limit 100MB
+      blobLimit: 500 * OneMB,
+      // total blob limit 100GB
+      storageQuota: 100 * OneGB,
+      // seat quota 20GB per seat
+      seatQuota: 20 * OneGB,
+      // history period of validity 30 days
+      historyPeriod: 30 * OneDay,
+      // member limit 1, override by workspace config
+      memberLimit: 1,
+      // copilot action limit 10
+      copilotActionLimit: 10,
```

Comment:
seems we should make copilot action limit optional, so let user fallback to ai quota

Reply:
in addition, the user quota should be separated from workspace quota.

user facing data types:
```
interface UserQuota {
  copilotActionLimit: number
}
```

```
// read team quota for team workspace
// read owner quota for normal workspace
interface WorkspaceQuota {
  blobLimit: number
  storageQuota: number
  memberLimit: number
  historyPeriod: number
}
```